scanspec/core.py

```diff
@@ -158,9 +160,9 @@ def deserialize(cls: Type[T], serialization: Mapping[str, Any]) -> T:
         return deserialize(cls, serialization)
 
 
-#: Map of positions keys to positions_ndarray
+#: Map of axes to points_ndarray
 #: E.g. {xmotor: array([0, 1, 2]), ymotor: array([2, 2, 2])}
-Positions = Dict[Any, np.ndarray]
+Points = Dict[Any, np.ndarray]
```

Comment:
```suggestion
AxesPoints = Dict[Any, np.ndarray]
```

Reply:
Suggestion to call this ScanPoints instead?

Reply:
I think I prefer AxesPoints as this is a dict {axis: points}

Reply:
Very well, changes made as suggested